gf-ui-lib/src/components/Button/Button.tsx

```diff
@@ -25,7 +29,7 @@ export const Button: React.FC<ButtonProps> = (props: ButtonProps) => {
   return (
     <button
       type={props.type ?? "button"}
-      className={styles.button}
+      className={`${styles.button} ${props.size === "content" ? styles.content : ""}`}
```

Comment:
Ternarian operatorn in template strings in render method in component is too hard to understand. Don't be shy to move out any logic from return. Make return as simple as possible

Reply:
Done